components/script/dom/document.rs

```diff
@@ -1234,11 +1234,39 @@ impl Document {
         }
     }
 
+    /// Return the title if one is specified
+    ///
+    /// https://html.spec.whatwg.org/multipage/#document.title
+    fn get_title(&self) -> Option<DOMString> {
```

Comment:
- I think this documentation comment could be a bit better and is missing a period.
- It's a rustdoc error to have a raw URL so this should be surrounded by angle brackets.
- Generally speaking, in Rust this should be named simply `title()`: https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter.

```suggestion
    /// Determine the title of the [`Document`] according to the specification at:
    /// <https://html.spec.whatwg.org/multipage/#document.title>. The difference
    /// here is that when the title isn't specified `None` is returned.
    fn title(&self) -> Option<DOMString> {
```

Reply:
Makes sense, regarding the name: since `Title` is already defined wouldn't another name be more appropriate? Feels like having both `title` and `Title` could be confusing.